packages/modules/src/modules/banners/designableBanner/DesignableBanner.tsx

```diff
@@ -214,6 +222,10 @@ const DesignableBanner: ReactComponent<BannerRenderProps> = ({
         );
     };
 
+    const showReminder =
+        (abTestName.includes('REMIND_ME_LATER') && abTestVariant === 'V1_SHOW_REMIND_ME_LATER') ||
```

Comment:
I don't think this bit is necessary - we can configure it from the RRCP with the secondary cta type

Reply:
We can but I think that'll make more work as the secondary CTA type puts a "remind me" button under the main CTA, I put this in so as not to alter that, as we discovered that it's in use on one banner that's currently live (or at least was when this was in planning). My intent was to have it this way and then to alter the way the "remind me" CTA looks if it becomes the new control, without making too many changes that have to be reversed if it doesn't

Reply:
oh right! Yes let's make sure we replace the old UI after this test